libcovebods/config.py

```diff
@@ -18,13 +18,49 @@
             "schema_url": os.path.join(_schema_folder, "schema-0-3-0.json"),
             "schema_url_host": _schema_folder,
         },
+        "0.4": {
+            "schema_url": os.path.join(_schema_folder, "schema-0-4-0"),
+            "schema_url_host": _schema_folder,
+        },
     },
     # In some cases we default to the latest schema version, so we need to know what the latest version is.
     # It should be a key that is in the 'schema_versions' data.
-    "schema_latest_version": "0.3",
+    # Note: Schema changed substantially from 0.3 to 0.4 (record based) so in cases where the schema
+    # version is not specified will try to determine whether is pre or post that change, and default to
+    # either the latest version (if after change) or a pre-change schema if before change
+    "schema_latest_version": "0.4",
+    # Or latest version before change to records (in 0.4)
+    "schema_latest_nonrecord_version": "0.3",
     # These default values are very wide on purpose. It is left to apps using this to tighten them up.
-    "bods_additional_checks_person_birthdate_min_year": 1,
+    "bods_additional_checks_person_birthdate_min_year": 1800,
```

Comment:
Should note this change in the changelog I think, so apps using this are prompted to check the new value works for them or to override it